src/main/webapp/WEB-INF/templates/search.jsp

```diff
@@ -12,7 +13,23 @@
     <t:base/>
     <div class="gcse-searchbox-only" data-resultsUrl="/search"></div>
     <div class="srp-container">
-      <div class="gcse-searchresults-only"></div>
+      <div class="srp-child">
+        <div class="gcse-searchresults-only"></div>
+      </div>
+      <div class="srp-child">
+        <c:forEach items='${requestScope["cardList"]}' var="knowledgeCard">
+          <div class="card">
+            <div class="card-content">
+              <a href="${knowledgeCard.getLink()}" title="card-source">
+                <span class="card-title">${knowledgeCard.getTitle()}</span>
+              </a>
+              <p>${knowledgeCard.getCode()}</p>
+              <span class="card-title">Description</span>
+              <p>${knowledgeCard.getDescription()}</p>
+            </div>
+          </div>
+        </c:forEach>
```

Comment:
this code doesn't seem to work when i combine all three client on my local sever. Each client works well with your precious code of getting single card.

Reply:
reference the comment left in #33